packages/utilities/psammead-storybook-helpers/README.md

```diff
@@ -4,9 +4,18 @@ This package provides a collection of common values that are used in storybook b
 
 ## Exports
 
-`LANGUAGE_VARIANTS` - A list of text samples in different languages.
+`LANGUAGE_VARIANTS` - A list of text samples in different languages, with the script and direction that should be used for that language.
 
-`inputProvider` - A function that provides support for selecting between the `text` and `select` storybook knobs. When using the `select` knob, users will be prompted to select a language, which will insert text from `LANGUAGE_VARIANTS` into the story.
+`inputProvider` - A function that provides support for previewing components in storybook in different languages. Takes two arguments, `slots` and `renderFn`. Sets the `dir` attribute on the `<html>` element in the story iframe using [Helmet](https://www.npmjs.com/package/react-helmet).
+
+- `slots`: `[{ name, defaultText }]` Required (but can be empty or null).
+  - `name`: String uniquely identifying this slot in the story. Required.
+  - `defaultText`: Default string to use when the story is showing English text. Optional.
+- `renderFn`: `(slotTexts, script, dir) => React.Component`
+  - `slotTexts`: Array of strings to insert into the story. Length and order corresponds to the provided `slots`.
+  - `script`: A [script](https://github.com/bbc/psammead/tree/latest/packages/utilities/gel-foundations#script-support) corresponding to the language selected by the storybook user.
+  - `dir`: Either `'ltr'` or `'rtl'`, corresponding to the language currently selected by the storybook user.
+  - Should return a react component.
```

Comment:
Given the syntax this should probably be the closing back-tick to `React.Component`

```suggestion
  ` Should return a react component.
```

Reply:
Sorry, I don't understand what you're suggesting/what's wrong

Reply:
The last hyphen `  - Should return a react component.` doesn't describe a key as the other hyphen do. Now that I've re-read it I think we could remove `- Should return a react component.`  and have line 14 read:
```
- `renderFn`: `(slotTexts, script, dir) => React.Component` Should return a react component.
```

Reply:
I rewrote this section while addressing your other comment; hopefully it reads better now